get-started/index.html

```diff
@@ -952,7 +953,7 @@ <h5>Data Modeling</h5>
             <dl class="accordion">
                <dd class="accordion-navigation inactive">
                   <a href="/developer/working-with-data/">
-                     <h5>Working with Data</h5>
+                     <h5>Data Import & Visualization</h5>
```

Comment:
I'd love to discuss this topic, going forward, for now this covers the content below.